manifests/init.pp

```diff
@@ -10,14 +10,16 @@
   Boolean $debug = false,
   Boolean $trace = false,
   Boolean $announce_cluster = false,
-  String $binpath = "/usr/sbin/gnatsd",
-  String $configdir = "/etc/gnatsd",
-  String $piddir = "/var/run",
+  Stdlib::Absolutepath $binpath = "/usr/sbin/gnatsd",
```

Comment:
There is something to do here afterall, we need to bump the minimum version requirements from `4.12.0` to `4.13.0` in the `metadata.json` since these types arrived then